### ISSUE TITLE
Connection Reuse - Use RFC2119 terminology

### ISSUE THREAD

**Issue by brettsheffield** [quicwg/base-drafts#2913]
> ## Connection Reuse 

> A server that does not wish clients to reuse connections for a particular origin can indicate that ...

`s/can/MAY/` - use RFC2119 terminology.

**Comment by LPardue** [comment]
This text is imported verbatim from RFC 7540 https://tools.ietf.org/html/rfc7540#section-9.1.1

I think it's fine as is because the internal linkage to  description of status 421 contains some normative text: https://tools.ietf.org/html/rfc7540#section-9.1.2

**Comment by MikeBishop** [comment]
Yeah, I'm not sold on this change.  This is expressing an option that servers have for that situation, not permission to undertake a new protocol behavior we're defining in this section.
